packages/calypso-analytics/global.d.ts

```diff
@@ -0,0 +1,7 @@
+export {};
+declare global {
+	interface Window {
+		_tkq: Array< Array< any > >;
```

Comment:
I believe it would be preferable to move this declaration to the module that adds `_tkq` to the window.

Reply:
It seems like _tkq could be initialized in a few different places, each place where it is used, the code will initialize it if it's not defined with: `window._tkq = window._tkq || [];` It looks like calypso-analytics *is* the primary place it's initialized because the other places are all apps.